src/index.ts

```diff
@@ -40,6 +40,45 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('logger', function(value: unknown): boolean {
+			console.log(value)
+			return true
+		})
+
+		hbs.registerHelper('isAnyOperationSupportMultipart', function(operations: CodegenOperation[]): boolean {
+			return Object.values(operations).some(op => op.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data')
+		})
+
+		hbs.registerHelper('isOperationSupportsMultipart', function(value: CodegenOperation): boolean {
+			return value.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data'
+		})
+
+		hbs.registerHelper('isContentSupportsMultipart', function(value: CodegenContent): boolean {
+			return value.mediaType.mimeType === 'multipart/form-data'
+		})
+
+		hbs.registerHelper('isMultipartSchema', function(value: CodegenSchema): boolean {
+			return value.nativeType.nativeType.endsWith('.MultipartFormData')
+		})
+
+		hbs.registerHelper('isMultipartSchemaPart', function(value: CodegenSchema): boolean {
+			return !!value.nativeType.serializedType.match('\\b.MultipartFormData.*Part$')
```

Comment:
I'd like to find a better way to do this... this is a weak connection that could break easily in future.